16/umbraco-cms/reference/developer-mcp/excluded-tools.md

```diff
@@ -0,0 +1,133 @@
+---
+description: The tools that have been excluded from the Developer MCP
+---
+
+# Excluded tools
+
+**⚠️ Intentionally Excluded:** 69 endpoints across 14 categories
```

Comment:
Maybe the numbers here created an unnecessary maintenance burden.  I'm sure you had some help in getting the counts the first time!  But if and when we add a new API endpoint, we wouldn't want to have to update the numbers here.